apps/phone/src/apps/contacts/components/views/ContactInfo.tsx

```diff
@@ -73,6 +141,16 @@ const ContactsInfoPage: React.FC = () => {
     updateContact({ id: contact.id, number, avatar, display: name });
   };
 
+  const openpayModal = () => {
+    if (ResourceConfig.general && ResourceConfig.general.useResourceIntegration && 
+        ResourceConfig.contacts && ResourceConfig.contacts.frameworkPay) {
```

Comment:
Couldn't you use option chaining here? Feel free to correct me if I'm wrong
`ResourceConfig?.general?.useResourceIntegration`

Reply:
You are correct, I meant to do this before pushing it but I guess I just moved on the next "to do"